test/math_test.rego

```diff
@@ -0,0 +1,28 @@
+package math_test
+
+import data.math
+
+test_ceil {
+	math.ceil(0) == 0
+	math.ceil(0.0000001) == 1
+	math.ceil(0.5) == 1
+	math.ceil(0.99999999) == 1
+	math.ceil(-0.00000001) == 0
+	math.ceil(-0.99999999) == 0
+}
+
+# Note: OPA considers 1 == 0.99999999999999999999
+# https://play.openpolicyagent.org/p/70Iqjgc24W
+# This seems like a bug but we'll ignore it for now
```

Comment:
I think there's an end to how precise we can be with floats. Not passing the bucket, but [the underlying Go lib](https://golang.org/pkg/math/big/#Float) that we use decides that it's the same: https://play.golang.org/p/UUrCZQuIowP

Reply:
Thanks for the background @srenatus - is this rounding error your Office Space style plan to richness? 😄 💸